src/Microsoft.AspNetCore.Sockets.Client.Http/WebSocketsTransport.cs

```diff
@@ -198,7 +208,7 @@ public WebSocketsTransport(ILoggerFactory loggerFactory)
             finally
             {
                 _logger.SendStopped(_connectionId);
-                _transportCts.Cancel();
+                TriggerCancel();
```

Comment:
Is this needed? if you get here it means that receiveCts has been cancelled or there was an exception. I think we only need `_transportCts.Cancel()` here.